diagnostics/convective_transition_diag/doc/convective_transition_diag.rst

```diff
@@ -24,26 +24,16 @@ The currently package consists of following functionalities:
 
 #. Convective Transition Basic Statistics (convecTransBasic.py)
 #. Convective Transition Critical Collapse (convecTransCriticalCollape.py)
-#. (\*) Moisture Precipitation Joint Probability Density Function (cwvPrecipJPDF.py)
-#. (\*) Super Critical Precipitation Probability (supCriticPrecipProb.py)
+#. (\*) Precipitation Contribution Function (cwvPrecipContrib.py)
 
 More on the way... (\* under development)
 
-As a module of the MDTF code package, all scripts of this package can be found under the `convective_transition_diag <https://github.com/NOAA-GFDL/MDTF-diagnostics/tree/main/var_code/convective_transition_diag>`__ directory and pre-digested observational data under ``inputdata/obs_data/convective_transition_diag``.
+As a module of the MDTF code package, all scripts of this package can be found under the `convective_transition_diag <https://github.com/NOAA-GFDL/MDTF-diagnostics/tree/main/diagnostics/convective_transition_diag>`__ directory and digested observational data under ``inputdata/obs_data/convective_transition_diag``.
 
 Required programming language and libraries
 -------------------------------------------
 
-The is package is written in Python 2, and requires the following Python packages:
-os, glob, json, Dataset, numpy, scipy, matplotlib, networkx, warnings, numba, & netcdf4. These Python packages are already included in the standard Anaconda installation.
-
-Known issue with matplotlib
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The plotting scripts of this POD may not produce the desired figures with the latest version of matplotlib (because of the default size adjustment settings). The matplotlib version comes with the Anaconda 2 installer, version 5.0.1 has been tested. The readers can switch to this older version.
-
-Depending on the platform and Linux distribution/version, a related error may occur with the error message "... ImportError: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory". One can find the missing object file ``libcrypto.so.1.0.0`` in the subdirectory ``~/anaconda2/pkgs/openssl-1.0.2l-h077ae2c_5/lib/``, where ``~/anaconda2/`` is where Anaconda 2 is installed. The precise names of the object file and openssl-folder may vary. Manually copying the object file to ``~/anaconda2/lib/`` should solve the error. 
-
+The is POD is written in Python 3.7, and requires the following Python packages: os, glob, json, Dataset, numpy, scipy, matplotlib, networkx, warnings, numba, & netcdf4. These dependencies are included in the python3_base environment provided by the automated installation script for the MDTF Framework. Note that running this POD outside the provided environment may result in figures different from the `samples <http://www.cgd.ucar.edu/cms/bundy/Projects/diagnostics/mdtf/mdtf_figures/MDTF_QBOi.EXP1.AMIP.001.save/convective_transition_diag/convective_transition_diag.html>`__.
```

Comment:
Should we host this sample at GFDL as well?

Reply:
That makes sense to me.